### ISSUE TITLE
CI status badge

### ISSUE THREAD

**Issue by daviddias** [ipfs-inactive/dev-team-enablement#38]
We need Jenkins CI badges so that we can replace the Jenkins, Circle and Appveyor on README

<img width="260" alt="image" src="https://user-images.githubusercontent.com/1211152/40069727-0cebcbf4-5864-11e8-8fe5-b9f707fa3496.png">

**Comment by VictorBjelkholm** [comment]
Yeah, thought about this before and started thinking that maybe we can have a service that concatenates the different statuses and merges it into a matrix of statuses (similar to how browserstack (or whatever we used to run browser tests on) did it). It'll basically be several svgs included in one. Made a initial prototype over at https://git.victor.earth/VictorBjelkholm/badge-renderer but has to be moved, improved and deployed to be useful.
